test/test.js

```diff
@@ -417,21 +417,6 @@ describe('Page', function() {
     }));
   });
 
-  describe('Page.injectFile', function() {
-    it('should work', SX(async function() {
-      const helloPath = path.join(__dirname, 'assets', 'injectedfile.js');
-      await page.injectFile(helloPath);
-      const result = await page.evaluate(() => __injected);
-      expect(result).toBe(42);
-    }));
-    it('should include sourcemap', SX(async function() {
```

Comment:
let's keep a test that checks for sourcemaps

Reply:
Missed that one, good catch!